src/interfaces/operation.ts

```diff
@@ -229,13 +216,21 @@ export interface OperationModalInvokeConfirm {
      */
     type: OperationPostInvokeConfirmType | OperationPreInvokeType | string
     /**
-     * Title for modal
+     * Body text of a modal
      */
     message: string
     /**
-     * Additional text for modal
+     * Custom modal title
```

Comment:
Well that's misleading all right. Probably TODO for renaming will be appropriate.

Reply:
todo is added